src/registrar/models/domain_request.py

```diff
@@ -589,11 +590,25 @@ def get_action_needed_reason_label(cls, action_needed_reason: str):
         verbose_name="last updated on",
         help_text="Date of the last status update",
     )
+
     notes = models.TextField(
         null=True,
         blank=True,
     )
 
+    def get_first_status_set_date(self, status):
```

Comment:
Just curious, would we expect any cases where we'd want to get the date a request was first given a certain status aside from `started`?

Reply:
Also don't have a strong opinion so I'm happy to go with whatever data from audit logs you think would be useful in the future!